.github/workflows/ci.yml

```diff
@@ -4,14 +4,24 @@ on: [push, pull_request]
 # for testing purposes. Will focus on main branch after thia workflow is integrated
 
 jobs:
-
-  lint:
+  build:
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        node-version: [10.19]
```

Comment:
This is a pretty old version of node. Not a deal breaker, but if board will only run on v10 now maybe we should create a ticket to find out why. I have v16 and I'm pretty sure I used v14 or 16 to build board but it's been a while. v10 hasn't had a security patch in at least a year so I'm not sure if it's even maintained anymore.

Reply:
Good call, I'll make a ticket.